docs/resource/k8s.md

```diff
@@ -91,6 +91,7 @@ containers on your cluster.
 <!-- semconv k8s.pod -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| `k8s.pod.labels.<key>` | string | The labels placed on the Pod, the `<key>` being the label name, the value being the label value. | `pod.labels.app=my-app,pod.labels.mycompany.io/arch=x64` | Recommended |
```

Comment:
Is Recommended the right requirement level? Adding all labels when available can lead to rather large events. Furthermore, since the key can be freely chosen, there can be a large number of different attributes creating sparse data sets. Maybe a lower requirement level would be more suitable?

Reply:
I personally am in favor of more metadata on the telemetry we produce, so I like this level.

Reply:
I tend to think that since labels are so prevalent in use in Kubernetes, it makes sense to have them as Recommended.